.gitignore

```diff
@@ -21,3 +21,5 @@ dist/
 .env.development.local
 .env.test.local
 .env.production.local
+
+.vercel
```

Comment:
While we at it.. can we sort the whole file (VS Code)

Reply:
What do you mean, just sort these lines alphabetically? The gitignore still has a `now` in there, though that probably doesn't need to be there anymore.

Reply:
yes.. we can get rid of now and sort the whole file alphabetically